src/detect-engine-siggroup.c

```diff
@@ -375,6 +378,24 @@ int SigGroupHeadClearSigs(SigGroupHead *sgh)
     return 0;
 }
 
+#ifdef __SSE2__
```

Comment:
could try adding 256 bits (AVX) or 512 bits (AVX512) variants here, however even in my expensive test it only spends 0.11% of the time here (vs 10% previously), so it's probably not worth the effort.